src/renderers/testing/ReactTestTextComponent.js

```diff
@@ -15,6 +15,7 @@
 import type { ReactText } from 'ReactTypes';
 
 class ReactTestTextComponent {
+  _currentElement: ReactText;
```

Comment:
Not sure if we've been doing it consistently, but it would be good to give a bit of whitespace between types and actual JS - can you just put a newline under here (and in the other class too)